package.json

```diff
@@ -39,5 +42,19 @@
     "typescript-eslint": "^8.36.0",
     "vite": "^7.0.4",
     "vitest": "^3.2.4"
+  },
+  "dependencies": {
+    "@types/react": "^19.1.8",
+    "@types/react-dom": "^19.1.6",
```

Comment:
issue (blocking): This should be in `devDependencies`

Reply:
Indeed, it should.